src/main/resources/db/migration/V43_47__Create_community_zone_table.sql

```diff
@@ -0,0 +1,7 @@
+create table community_zone
+(
+    id   varchar primary key default uuid_generate_v4(),
+    name varchar not null,
```

Comment:
If you wanted to explicit the community avalaible zone, and it's geo-jobs (us) that decides what zone be can used, the best way is to set available_zone_name as type ENUM.